core/src/apps/ethereum/keychain.py

```diff
@@ -44,16 +44,24 @@
     paths.PATTERN_BIP44,
     paths.PATTERN_SEP5,
     paths.PATTERN_SEP5_LEDGER_LIVE_LEGACY,
+    paths.PATTERN_CASA,
 )
 
 
 def _schemas_from_address_n(
     patterns: Iterable[str], address_n: paths.Bip32Path
 ) -> Iterable[paths.PathSchema]:
-    if len(address_n) < 2:
+    # Exclude Casa export paths (m/45')
```

Comment:
i don't think this should be here anymore?

Reply:
Good catch

Reply:
Fixed